Vivekananda_session1_4VP20CS062_Prajwal/hw.py

```diff
@@ -0,0 +1,12 @@
+import requests
+import csv
+from bs4 import BeautifulSoup
+res=requests.get("https://blog.python.org/")
+soup = BeautifulSoup(res.content, "html.parser")
+titles = soup.find_all("h1")
+with open('output2.csv','w',newline='') as f:
```

Comment:
@Prajwal7Amin Any reason why you're first writing to a CSV file and then writing the contents of the CSV file to database instead of directly writing to the database?

Reply:
"@anushds I tried to store data directly into the database without encountering any errors. However, upon checking the contents of the table, I found that it was empty. Therefore, I  used a this approach, which worked successfully."

Reply:
@Prajwal7Amin 
Alright, got it. But I would suggest that whenever you reach a dead end where you think it doesn't work(after certain tries), just have the things you tried in the commits of your PR, so that the mentors can see(by checking that specific commit) where you went wrong in that approach and correct your approach.